src/app/globals.css

```diff
@@ -1,3 +1,13 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* doesnt seem to work */
```

Comment:
That was for styling elements created by maplibre (the underlying map library used by react-map-gl) that can't be styled through react, like improving the look of the default popup and hiding the "attribution" thing that is always displayed in the bottom right corner of the map. But they don't seem to applied at all even though im pretty sure the selectors are correct, so thought it might be some tailwind-specific issue related to how global styles work (?), but couldn’t figure it out. Kept it because I was planning to come back to it later at some point